examples/cloudml-sklearn-pipeline/config/config.yaml

```diff
@@ -0,0 +1,4 @@
+trainingInput:
+  scaleTier: STANDARD_1
+  runtimeVersion: "1.13"
+  pythonVersion: "3.5"  # Note: Python 3 is also supported
```

Comment:
Since you are using python 3, I do not think you need this note in the files

Reply:
I have modified as "# Note: Both Python 2.7 and Python 3 is also supported"